docs/installing-open-mpi/configure-cli-options/networking.rst

```diff
@@ -41,7 +41,7 @@ can be used with ``configure``:
   compiler/linker search paths.
 
   Libfabric is the support library for OpenFabrics Interfaces-based
-  network adapters, such as Cisco usNIC, Intel True Scale PSM, etc.
+  network adapters, such as Cisco usNIC, Cornelis Networks True Scale PSM, etc.
```

Comment:
Since PSMv1 (as opposed to PSM support in general, nevermind that PSM2 and PSM3 are different things) support is no longer in OMPI, can remove "Cornelis Networks True Scale PSM".